SignalServiceKit/src/Storage/OWSPrimaryStorage.h

```diff
@@ -23,6 +25,7 @@ extern NSString *const OWSUIDatabaseConnectionNotificationsKey;
 @property (nonatomic, readonly) YapDatabaseConnection *uiDatabaseConnection;
 @property (nonatomic, readonly) YapDatabaseConnection *dbReadConnection;
 @property (nonatomic, readonly) YapDatabaseConnection *dbReadWriteConnection;
+@property (nonatomic) NSSet<LKDeviceLink*> *deviceLinkCache;
```

Comment:
You don't need to do this. You can just make it a static variable on our extension of `OWSPrimaryStorage`. That way we don't mix Swift and Obj-C and we keep our code separate from the core Signal code.